src/App.js

```diff
@@ -575,7 +575,10 @@ NavigationService.setTopLevelNavigator(navigationRef);
 const App = () => (
   <SafeAreaProvider>
     <Provider store={store}>
-      <SafeAreaView style={{ flex: 1 }}>
+      <SafeAreaView
+        edges={['top', 'right', 'left']}
```

Comment:
Avoids rendering on the bottom margin, as it consumes significant space in return for a user experience that does not seem to be improved.